packages/dds/tree/src/test/simple-tree/objectNode.spec.ts

```diff
@@ -121,4 +122,18 @@ describe("ObjectNode", () => {
 		const keys = Object.keys(n);
 		assert.deepEqual(keys, ["foo"]);
 	});
+
+	it("test array", () => {
```

Comment:
Probably not the right test name :)

Reply:
oops, copied that for the work item, and yes, that needs updating.

```suggestion
	it("delete operator", () => {
```